test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1047,9 +1047,13 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				}
 				sort.Ints(coreids)
 				Expect(len(coreids)).ToNot(Equal(0))
-				middleCoreIds := coreids[len(coreids)/2]
-				coresiblings := nodes.GetAndRemoveCpuSiblingsFromMap(numaCoreSiblings, middleCoreIds)
-				reserved = reserved.Union(coresiblings)
+				middleIndex := len(coreids) / 2
+				// we need at least 4 cpus for reserved cpus
```

Comment:
how do we know we can afford 4 reserved cores and then there will be enough isolated cores?

Reply:
We have a condition above to check if we have more than 8 cpus to run the tests:
https://github.com/openshift/cluster-node-tuning-operator/blob/d2c881fb8e2a08516885455f8024c4bcaacf1d5a/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go#L1033

Reply:
ok then

Reply:
So the tests from here are skipped in all prow CI runs? Afaik we only have 4 cpus available in Prow, right?